main.go

```diff
@@ -22,6 +22,7 @@ var Options struct {
 	HTTPSCAFile           string `envconfig:"HTTPS_CA_FILE"`
 	ListenPort            string `envconfig:"LISTEN_PORT" default:"8080"`
 	RequestAuthType       string `envconfig:"REQUEST_AUTH_TYPE"`
+	MaxConcurrentRequests int    `envconfig:"MAX_CONCURRENT_REQUESTS" default:"400"`
```

Comment:
Should we hardcode this for the operator deployment or leave the default?

Reply:
I think it's fine for now. Previously it was not limited at all, now we have a knob.

400 was around the most I could get to succeed running the image service locally on my laptop so I doubt we'll be able to do more in a real deployment.